### ISSUE TITLE
Storyshots with Vue Not Working Properly

### ISSUE THREAD

**Issue by derekshull** [storybooks/storybook#2797]
### Issue details
I'm attempting to use Storyshots with Nuxt.js (Vue).  I'm trying to use the same set up as in the [example here](https://github.com/storybooks/storybook/blob/master/examples/vue-kitchen-sink/vueshots.test.js).  I keep getting the following error: `storyshots is intended only to be used with storybook`.  I've tried several things over the last night (all night actually 🤦‍♂️ ) to try getting it work but can't figure out why the error is occuring.  Here's the full output I get:

```
> jest --config test/unit/jest.conf.js --coverage

 PASS  test/unit/specs/components/Button.spec.js
 PASS  test/unit/specs/pages/index.spec.js
 PASS  test/unit/specs/components/Footer.spec.js
 PASS  test/unit/specs/layouts/default.spec.js
 FAIL  test/unit/storyshots.test.js
  ● Test suite failed to run

    storyshots is intended only to be used with storybook

      2 | import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';
      3 |
    > 4 | initStoryshots({
      5 |   framework: 'vue',
      6 |   configPath: path.join(__dirname, '.storybook'),
      7 |   test: multiSnapshotWithOptions({}),

      at testStorySnapshots (node_modules/@storybook/addon-storyshots/dist/index.js:127:11)
      at Object.<anonymous> (test/unit/storyshots.test.js:4:31)

Test Suites: 1 failed, 4 passed, 5 total
Tests:       4 passed, 4 total
Snapshots:   4 passed, 4 total
Time:        4.447s
Ran all test suites.
--------------|----------|----------|----------|----------|----------------|
File          |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
--------------|----------|----------|----------|----------|----------------|
All files     |    11.76 |        0 |    18.18 |    12.12 |                |
 components   |       50 |      100 |       50 |      100 |                |
  Button.vue  |       50 |      100 |       50 |      100 |                |
 layouts      |       10 |        0 |        0 |       10 |                |
  default.vue |      100 |      100 |      100 |      100 |                |
  error.vue   |        0 |        0 |        0 |        0 |... 15,17,19,39 |
 pages        |     9.09 |        0 |     12.5 |     9.09 |                |
  _id.vue     |        0 |        0 |        0 |        0 |... 37,39,41,61 |
  index.vue   |       50 |        0 |       50 |       50 |           6,10 |
--------------|----------|----------|----------|----------|----------------|
npm ERR! Test failed.  See above for more details.
```
I am able to use Jest to run some tests, which are the tests that you see passing in the output above (You can find the 4 tests that are using Jest in `test/unit/specs`).

### Steps to reproduce
You can find [the files I'm using here](https://github.com/derekshull/nuxt-starter-kit-v2/tree/master/test/unit).  The stories are [located here](https://github.com/derekshull/nuxt-starter-kit-v2/tree/master/components).

To reproduce it you can [clone the repo](https://github.com/derekshull/nuxt-starter-kit-v2), run `npm install`, launch Storybook by running `npm run storybook`, then run the `npm test` to start the tests to get the same output as I'm getting in the above section.

### Please specify which version of Storybook and optionally any affected addons that you're running

- `@storybook/vue` 3.3.10
- `@storybook/addon-storyshots` 3.3.10

**Comment by igor-dv** [comment]
Storyshots addon for vue was released in 3.4.0-alpha.5

**Comment by derekshull** [comment]
Ah, the documentation made it sound like it was already available in the stable version. Thanks for letting me know!
